src/foam/dao/EasyDAO.js

```diff
@@ -305,11 +305,6 @@ return delegate;
       name: 'authorize',
       value: false
     },
```

Comment:
can you add property so we can set authorizer on EasyDAO. If set then that's the authorizer we set to AuthorizationDAO, otherwise standard/authorizable

Reply:
the authorizer property was added in previous authorization pr https://github.com/foam-framework/foam2/pull/2418/files